pkg/platform/abstract/platform.go

```diff
@@ -1762,15 +1762,6 @@ func (ap *Platform) enrichTriggers(ctx context.Context, functionConfig *function
 			}
 		}
 
-		if triggerInstance.WorkerAllocatorName == "" {
-			triggerInstance.WorkerAllocatorName = triggerName
-			ap.Logger.DebugWithCtx(ctx, "Enriched WorkerAllocatorName",
-				"triggerName", triggerName,
-				"triggerKind", triggerInstance.Kind,
-				"WorkerAllocatorName", triggerInstance.WorkerAllocatorName,
-			)
-		}
-
```

Comment:
Technically, only this part should be reverted, so I'll create a PR with filtering changes as soon as we merge this PR